app/controllers/posts_controller.rb

```diff
@@ -370,11 +370,18 @@ def editor_setup
   end
 
   def import_thread
-    begin
-      importer = PostImporter.new(params[:dreamwidth_url])
-      importer.import(params[:board_id], current_user.id, section_id: params[:section_id], status: params[:status], threaded: params[:threaded])
-    rescue PostImportError => e
-      flash.now[:error] = e.api_error
+    importer = PostImporter.new(params[:dreamwidth_url])
```

Comment:
Should this be `import_params[:dreamwidth_url]`? Or should we remove `:dreamwidth_url` from the allowed list below (since we're just skipping it, and I'm assuming we don't use it elsewhere)?